dashboard/src/components/diagram/right-sidebar/proto-components/nodes/user-task.tsx

```diff
@@ -0,0 +1,36 @@
+import { Section } from "../../section";
+import { Label } from "../../label";
+import { UserTaskNode } from "littlehorse-client/proto";
+import { getVariable } from "@/utils/data/variables";
+import { TaskNodeComponent } from "./task";
+import { TaskNode } from "littlehorse-client/proto";
+import VariableMutationComponent from "../variable-mutation";
+
+export function UserTaskNodeComponent(userTask: UserTaskNode) {
+  return (
+    <Section title="UserTaskNode">
+      <Label label="UserTaskDefName">{userTask.userTaskDefName}</Label>
+      {userTask.userGroup && <Label label="UserGroup">{getVariable(userTask.userGroup)}</Label>}
+      {userTask.userId && <Label label="UserId">{getVariable(userTask.userId)}</Label>}
+      {userTask.userTaskDefVersion && <Label label="UserTaskDefVersion">{userTask.userTaskDefVersion}</Label>}
+      {userTask.notes && <Label label="Notes">{getVariable(userTask.notes)}</Label>}
+      {userTask.onCancellationExceptionName && <Label label="OnCancellationExceptionName">{getVariable(userTask.onCancellationExceptionName)}</Label>}
+      <Section title="Actions">
+        {userTask.actions.map((action, index) => (
+          <Section key={JSON.stringify(action) + index} title={`UTActionTrigger ${index + 1}`}>
+            {action.task && <Section title="UTATask">
+              <TaskNodeComponent {...(action.task.task as TaskNode)} />
+              <Section title="Mutations">
+                {action.task.mutations.map((mutation, index) => (
+                  <VariableMutationComponent key={JSON.stringify(mutation) + index} {...mutation} />
```

Comment:
Why `JSON.stringify` for the component key?

Reply:
Looks like what you're having is duplicated keys because you were assigning just the index as key which could return a warning regarding reusing the same key for multiple components in a stacked map.

Would be better if you prefix the id with something, not the full json fo the action, otherwise the component size would be big just because.

do something like `key={"actions-${actionIndex}"}` and `key={"mutation-${actionIndex}-${mutationIndex}"}`

Rename the `index` to `actionIndex` and `mutationIndex` so they don't get shadowed.

Reply:
I see new docs are okay with stringify